wazuh/wazuh-kibana-app
# 4958 fix commands in the deploy new agent section

### Description

Fixed commands in the deploy new agent section (most of the commands are missing '-1').

### Issues Resolved
[List any issues this PR will resolve]

### Evidence

<img width="879" alt="image" src="https://user-images.githubusercontent.com/99441266/205712545-ef1538c7-6455-4611-9d02-786a8cacdf45.png">

**Steps to test**
1. Click on Agents
2. Click on Deploy new agent
3. See installation commands (all commands have the '-1).

### Check List
